prow/cmd/checkconfig/main.go

```diff
@@ -397,3 +404,21 @@ func validateNeedsOkToTestLabel(configAgent config.Agent) error {
 	}
 	return errorutil.NewAggregate(queryErrors...)
 }
+
+func verifyOwnersPlugin(agent plugins.PluginAgent) error {
+	// we do not know the set of repos that use OWNERS, but we
+	// can get a reasonable proxy for this by looking at where
+	// the `approve' plugin is enabled
+	ownersConfig := newOrgRepoConfig(agent.Config().EnabledReposForPlugin(approve.PluginName))
+	validateOwnersConfig := newOrgRepoConfig(agent.Config().EnabledReposForPlugin(verifyowners.PluginName))
+	var invalid []string
+	for _, target := range ownersConfig.orgs.Union(ownersConfig.repos).UnsortedList() {
+		if !validateOwnersConfig.has(target) {
```

Comment:
This logic will need to change a bit after https://github.com/kubernetes/test-infra/pull/9588 merges.

Reply:
I'm crossing my fingers that I win this race :grimacing:

Reply:
Lets not merge this feature yet if we know its going to need to be immediately updated. #9588 should be ready to merge, it has been waiting for review for a few days...